tests/test_crs/valid_uperf.yaml

```diff
@@ -3,7 +3,14 @@ kind: Benchmark
 metadata:
   name: example-benchmark
 spec:
-  cleanup: false
+  es:
+    server: elk-b09-h30-r720xd.rdu.openstack.engineering.redhat.com
```

Comment:
This is the test cr, so it can stay I believe

Reply:
My issue is more about the hostname, no the CR.

Reply:
I'm trying to come up with a solution for the CI to use environment vars as well, so this can also be eliminated soon. For now, I can remove it if you think its best.